### ISSUE TITLE
Cannot generate even the demo website

### ISSUE THREAD

**Issue by mattwyskiel** [jekyll-octopod/jekyll-octopod#39]
Hello,

I have just downloaded jekyll-octopod and followed all the instructions on the website. However, when I try to perform `octopod serve` for the first time, it gives me this error:

```
Configuration file: /Users/matthewwyskiel/Documents/Developer/Web/King Matt's A List/king-matts-a-list/_config.yml
            Source: /Users/matthewwyskiel/Documents/Developer/Web/King Matt's A List/king-matts-a-list
       Destination: /Users/matthewwyskiel/Documents/Developer/Web/King Matt's A List/king-matts-a-list/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Liquid Exception: undefined method `[]' for nil:NilClass in /_layouts/with_twitter_card.html
/Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-octopod-0.9.1/lib/jekyll/octopod_filters.rb:59:in `audio': undefined method `[]' for nil:NilClass (NoMethodError)
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/strainer.rb:56:in `invoke'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/context.rb:86:in `invoke'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/variable.rb:78:in `block in render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/variable.rb:76:in `each'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/variable.rb:76:in `inject'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/variable.rb:76:in `render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:109:in `render_node'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:88:in `block in render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:75:in `each'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:75:in `render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/template.rb:208:in `block in render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/template.rb:242:in `with_profiling'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/template.rb:207:in `render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/liquid-4.0.0/lib/liquid/template.rb:220:in `render!'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/liquid_renderer/file.rb:30:in `block (2 levels) in render!'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/liquid_renderer/file.rb:42:in `measure_bytes'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/liquid_renderer/file.rb:29:in `block in render!'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/liquid_renderer/file.rb:49:in `measure_time'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/liquid_renderer/file.rb:28:in `render!'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/renderer.rb:124:in `render_liquid'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/renderer.rb:189:in `render_layout'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/renderer.rb:156:in `place_in_layouts'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/renderer.rb:86:in `render_document'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/renderer.rb:63:in `run'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:455:in `block (2 levels) in render_docs'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:453:in `each'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:453:in `block in render_docs'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:452:in `each'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:452:in `render_docs'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:193:in `render'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/site.rb:74:in `process'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/command.rb:28:in `process_site'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/commands/build.rb:65:in `build'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/commands/build.rb:36:in `process'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/commands/serve.rb:42:in `block (3 levels) in init_with_program'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/commands/serve.rb:42:in `each'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/lib/jekyll/commands/serve.rb:42:in `block (2 levels) in init_with_program'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-3.6.0/exe/jekyll:15:in `<top (required)>'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-octopod-0.9.1/bin/octopod:272:in `load'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/gems/jekyll-octopod-0.9.1/bin/octopod:272:in `<top (required)>'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/bin/octopod:23:in `load'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0/bin/octopod:23:in `<main>'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0@global/bin/ruby_executable_hooks:15:in `eval'
        from /Users/matthewwyskiel/.rvm/gems/ruby-2.4.0@global/bin/ruby_executable_hooks:15:in `<main>'
```

Note: I am using the latest version of Jekyll, since my computer (i.e. bundler) refuses to install anything earlier than 3.6.0 (I've literally tried everything to downgrade, and nothing works).

EDIT: I found a way to ensure Jekyll 3.3.1, and still get the same exact error. Please help!!

**Comment by mattwyskiel** [comment]
It turns out I needed to delete the extraneous post from the original jekyll template site (from before I ran `octopod setup`

**Comment by parolkar** [comment]
I faced the same issue,
Had to delete the default post created by jekyll   under (_posts)

**Comment by marksweiss** [comment]
I submitted a PR to fix this. https://github.com/jekyll-octopod/jekyll-octopod/pull/43/files.

Thanks!

**Comment by haslinger** [comment]
@marksweiss  Thank you!
-> Is now fixed in master.
-> Will hopefully create a new release next week (have to update some docs before)  and then close this issue here.

**Comment by haslinger** [comment]
Well no ;-) The changes in the docs are unrelated, so new version created and published: 0.9.4.
